docs/src/installing-deis/installing-the-deis-platform.md

```diff
@@ -24,6 +24,7 @@ If you see a list of targets like the one above, helm can communicate with the k
 
 Once finished, run this command to provision the Deis platform:
 
+    $ helm update
```

Comment:
`helm repo add` fails due to the directory structure within ~/.helm/ being nonexistent. :smile: